vassal-doc/src/main/readme-referencemanual/ReferenceManual/GlobalOptions.adoc

```diff
@@ -34,7 +34,11 @@ It is recommended that for new modules you set this option to _Always_, as the o
 
 *Send-to-Location trait generates Movement Trails:*:: If this option is enabled, Send-to-Location traits generate Movement Trails (and mark pieces moved) when they cause a piece to move, assuming the piece in question has a Movement Trail trait of course.
 
-*Icons and hotkeys:*:: You can specify your own button icons and keyboard shortcuts for the logfile step/undo buttons and the button that shows/hides the server controls.
+[#leadingzeros]
+*Store integers with leading zeros as Strings internally:*::  This option should not normally be used. It is for the specific case where you use leading zeros in Grid co-ordinates and need to maintain these leading zeros for later reporting. By default, Vassal will strip leading zeros off and convert to an integer. Enabling this option forces Vassal to keep the leading zeros and store the number 02 as the string "02", not the number 2.
```

Comment:
Might not be worth changing, but I was thinking "Preserve leading zeros in Integers" would be a better indicator to module editors of what the option actually does than "Store integers with leading zeros as Strings internally".

Reply:
Yes indeed!